blog/electron-29-0.md

```diff
@@ -0,0 +1,128 @@
+- Upgraded Node from v18.18.2 to v20.9.0 [#40545](https://github.com/electron/electron/pull/40545)
+
+### Breaking Changes
+
+#### Behavior Changed: `ipcRenderer` can no longer be sent over the `contextBridge`
+
+Attempting to send `ipcRenderer` as an object over the `contextBridge` will now result in
+an empty object on the receiving side of the bridge. This change was made to remove / mitigate
+a security footgun, you should not directly expose ipcRenderer or it's methods over the bridge.
+Instead provide a safe wrapper like below:
+
+```js
+contextBridge.exposeInMainWorld('app', {
+  onEvent: (cb) => ipcRenderer.on('foo', (e, ...args) => cb(args)),
+});
+```
+
+#### Removed: `renderer-process-crashed` event on `app`
+
+The `renderer-process-crashed` event on `app` has been removed.
+Use the new `render-process-gone` event instead.
+
+```js
+// Removed
+app.on('renderer-process-crashed', (event, webContents, killed) => {
+  /* ... */
+});
+
+// Replace with
+app.on('render-process-gone', (event, webContents, details) => {
+  /* ... */
+});
+```
+
+#### Removed: `crashed` event on `WebContents` and `<webview>`
+
+The `crashed` events on `WebContents` and `<webview>` have been removed.
+Use the new `render-process-gone` event instead.
+
+```js
+// Removed
+win.webContents.on('crashed', (event, killed) => {
+  /* ... */
+});
+webview.addEventListener('crashed', (event) => {
+  /* ... */
+});
+
+// Replace with
+win.webContents.on('render-process-gone', (event, details) => {
+  /* ... */
+});
+webview.addEventListener('render-process-gone', (event) => {
+  /* ... */
+});
+```
+
+#### Removed: `gpu-process-crashed` event on `app`
+
+The `gpu-process-crashed` event on `app` has been removed.
+Use the new `child-process-gone` event instead.
+
+```js
+// Removed
+app.on('gpu-process-crashed', (event, killed) => {
+  /* ... */
+});
+
+// Replace with
+app.on('child-process-gone', (event, details) => {
+  /* ... */
+});
+```
+
+## End of Support for 26.x.y
+
+Electron 26.x.y has reached end-of-support as per the project's [support policy](https://www.electronjs.org/docs/latest/tutorial/electron-timelines#version-support-policy). Developers and applications are encouraged to upgrade to a newer version of Electron.
+
+| E29 (Feb'24) | E30 (Apr'24) | E31 (Jun'24) |
+| ------------ | ------------ | ------------ |
+| 29.x.y       | 30.x.y       | 31.x.y       |
+| 28.x.y       | 29.x.y       | 30.x.y       |
+| 27.x.y       | 28.x.y       | 29.x.y       |
+
+## What's Next
+
+In the short term, you can expect the team to continue to focus on keeping up with the development of the major components that make up Electron, including Chromium, Node, and V8.
+
+You can find [Electron's public timeline here](https://www.electronjs.org/docs/latest/tutorial/electron-timelines).
+
+More information about future changes can be found on the [Planned Breaking Changes](https://github.com/electron/electron/blob/main/docs/breaking-changes.md) page.
```

Comment:
I'd like to add a section on the RFC process here. I know it's not directly related to this release in particular but it'd be relevant for future releases! 

```suggestion
## What's Next

Did you know that Electron recently added a community Request for Comments (RFC) process? If you want to add a feature to the framework, RFCs can be a useful tool to start a dialogue with maintainers on its design. You can also see upcoming changes being discussed in the Pull Requests. To learn more, check out our [Introducing electron/rfcs](https://www.electronjs.org/blog/rfcs) blog post, or check out the README of the [electron/rfcs](https://www.github.com/electron/rfcs) repository directly.

In the short term, you can expect the team to continue to focus on keeping up with the development of the major components that make up Electron, including Chromium, Node, and V8.

You can find [Electron's public timeline here](https://www.electronjs.org/docs/latest/tutorial/electron-timelines).

More information about future changes can be found on the [Planned Breaking Changes](https://github.com/electron/electron/blob/main/docs/breaking-changes.md) page.
```